### PR TITLE
chore: add korean readme translation

### LINKED ISSUE #3580: 📎 Tranlate README.md to Korean
### Description

I'm trying to translate README into Korean. Can I just proceed?

### PR DESCRIPTION
**Pull request by Procrustes5**
## Summary
Added Korean README Translation!

## Ticket
closes #3580

### CODE REVIEW

**Comment by Conaclos** [comment]
Hi [Procrustes5](https://github.com/Procrustes5)! Thanks for your contribution :)

Have you some Korean reviewers to suggest?

File: packages/@biomejs/biome/README.kr.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+[हिन्दी](https://github.com/biomejs/biome/blob/main/packages/%40biomejs/biome/README.hi.md) | [English](https://github.com/biomejs/biome/blob/main/packages/%40biomejs/biome/README.md) | [简体中文](https://github.com/biomejs/biome/blob/main/packages/%40biomejs/biome/README.zh-CN.md) | [日本語](https://github.com/biomejs/biome/blob/main/packages/%40biomejs/biome/README.ja.md) | [Português do Brasil](https://github.com/biomejs/biome/blob/main/packages/%40biomejs/biome/README.pt-br.md) | 한글
+
+</div>
+
+
+**Biome** 은 웹 프로젝트를 위한 고성능 툴체인으로, 프로젝트를 건전하게 유지하기 위한 개발자 툴을 제공하는 것을 목표로 하고 있습니다.
+
+**Biome** 은 _JavaScript_, _TypeScript_, _JSX_ 그리고 _JSON_ 을 위한 **[고속 Formatter](./benchmark#formatting)** 로, **[_Prettier_ 와의 호환성 97%](https://console.algora.io/challenges/prettier)** 을 달성하고 있습니다.
+
+**Biome** 은 _JavaScript_, _TypeScript_, _JSX_을 위한 **[고성능 Linter](https://github.com/biomejs/biome/tree/main/benchmark#linting)** 로、ESLint, typescript-eslint, [등의 리소스](https://github.com/biomejs/biome/discussions/3)에서 **[170개 이상의 룰](https://biomejs.dev/linter/rules/)** 을 제공하고 있습니다. Biome 은 **상세하며 문맥에 맞는 결과를 출력**하기 위해, 코드를 개선하고, 더 좋은 프로그래머가 되기 위한 도움을 드립니다!
```
Comment (merri4) [inline packages/@biomejs/biome/README.kr.md:44]: This line should update the number `170` to `200` to follow the recent readme version.

File: packages/@biomejs/biome/README.kr.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+**Biome** 은 모던한 웹개발에 대한 [모든 주요 언어][language-support]를 지원하는 것을 목표로 합니다.
+
+**Biome** 이 동작하는 데에 Node.js 는 필요하지 않습니다.
+
+**Biome** 은 소스 코드의 완전한 표현력과 에러 회피 능력을 가진, 세련된 Parser 에 의해 우수한 LSP 지원을 제공합니다.
+
+**Biome** 은 지금까지 서로 다른 툴로 제공하던 기능들을 통합합니다. 공통된 기반을 구축하는 것으로 코드 처리, 에러 표시, 병렬 처리, 캐시, 설정에 대해 일관된 경험을 제공합니다.
+
+관심이 있는 분은 [프로젝트 이념][biome-philosophy] 을 확인해주세요.
```
Comment (merri4) [inline packages/@biomejs/biome/README.kr.md:89]: Rather than `프로젝트 이념`, `프로젝트 철학` looks more natural.

**Review by merri4 — COMMENTED** [review]
Suggesting few word changes. All other translations looks fine!

**Comment by Procrustes5** [comment]
@merri4 
Thank you for review!
I've changed my code, plz check.

File: packages/@biomejs/biome/README.kr.md (hunk 1 above)
Comment (Conaclos) [reply packages/@biomejs/biome/README.kr.md]: I think we can even write `260` :)

**Review by ematipico — APPROVED** [review]

File: packages/@biomejs/biome/README.kr.md (hunk 1 above)
Comment (Procrustes5) [reply packages/@biomejs/biome/README.kr.md]: Wow, that's really amazing👀 I reflected this!

**Review by merri4 — APPROVED** [review]
Everything looks good! Thanks for the translation work.

**Pull request merged**